Server/VikingVault.DataAccess/Models/UserProfilePageViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VikingVault.DataAccess.Models
+{
+    public class UserProfilePageViewModel
+    {
+        public int Id { get; set; }
+        public string Firstname { get; set; }
```

Comment:
This proprty does not comply with the C# naming conventions. Should be FirstName instead.